lib/utils/constants.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Constants
+ */
+module.exports = {
+  // Caret string for cursor positioning
+  CARET: Math.random().toString(36)
```

Comment:
Not sure we should use `Math.random()` here. This would break if the system ends up with many version of inquirer installed and some plugins are relying on another version.

Let's hardcode a random string here instead.

Reply:
good to know, not sure if I completely understand the issue with multiple versions.

Reply:
To explain the issue with multiple versions, it's because with nested modules systems like Node, there's no guarantee there's a single version of Inquirer installed.

So each version get its own instance. So things like randoms values or `instanceof` checks on classes breaks between different instances. And these issues are really hard to debug on the user side as it's hard to see - and just think about. (Had an issue like this when we first implemented the `inquirer.separator` for lists)

Hope this makes a bit a sense :)